tests/magic_bullet_analyzer_test.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Magic bullet analyzer tests.
+"""
+from analyzer.engines import magic_bullet_analyzer
+import re
+from spacy.en import English
```

Comment:
You don't need to import spacy for this test (you're already importing it in your magic_bullet_analyzer module).